File: lab5.cpp
#include<iostream>

using namespace std;

int main(){

int a=0,b=0;

cout<<"a=";
cin>>a;
cout<<"b=";
cin>>b;

cout << "p=";

for(int p=a;p<=b;p++){
 
int cnt=0;
    for(int i=2; i<=p/2; ++i){
        if(p%i==0)
           cnt++;}
      if(!cnt)
         cout<<p<<", ";
   }
   return 0;
}

File: lab5.py
a = float(input("a = "))
b = float(input("b = "))
p = a
print("p= ")
while (p<=b):
   
    cnt = 0
    j=2
    while(j<=p/2):
        if (p%j==0):
            
            cnt=cnt+1
        j=j+1
    p=p+1
    if (not cnt):
        print(p-1)